packages/cloud-cognitive/src/components/Card/Card.js

```diff
@@ -84,7 +85,13 @@ export let Card = ({
             />
           );
         return (
-          <div key={id} className={`${pkg.prefix}-card-icon`}>
+          <div
+            key={id}
+            className={`${pkg.prefix}-card-icon`}
+            onClick={onClick}
+            onKeyDown={onClick}
```

Comment:
Buttons are clicked on return key presses (repeats) and space (key up only).

Presumably here if I press the tab or escape it triggers onClick?

Reply:
You raise a good point. I honestly haven't given it that much thought. I'll ask Josh Black what Carbon's been doing for keyboard controls

Reply:
@lee-chase i may have been overthinking this when i commented this. While you might be correct about how `onKeyDown` works, it will ultimately be up to the consumer to figure out how they want their `onKeyDown` event to fire. I went ahead an made an update to add additional keyboard support, so now `onKeyDown` has it's own prop for specific `onKeyDown` implementation.